src/client/Microsoft.Identity.Client/AppConfig/PublicClientApplicationBuilder.cs

```diff
@@ -346,6 +347,8 @@ public PublicClientApplicationBuilder WithKerberosTicketClaim(string servicePrin
         /// <item><description>In mobile apps, the device must be Intune joined and Authenticator or Company Portal must be installed. See https://aka.ms/msal-brokers </description></item>
         /// </list>
         /// </remarks>
+        [Obsolete("This method is obsolete. Applications should rely on the library automatically falling back to a browser if the broker is not available. ", false)]

```

Comment:
This has the potential to brake some customers who use code analysis rulesets that prevent Obsolete apis from being used. Maybe it is a good idea to start highlighting our potentially breaking changes/deprications somewhere in our wiki/readme so that customers are aware?

@bgavrilMS

Reply:
Good point for release manager. Attn: @localden